docs/reference/security/sts-login.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Secure Token Service (STS) Authentication for lakeFS
+description: Authenticate with lakeFS using Secure Token Service (STS) by leveraging a remote authenticator. This feature enables integration with Identity Providers (IdPs) for secure and efficient user authentication.
+grand_parent: Reference
+parent: Security
+redirect_from:
+  - /reference/remote-authenticator.html
+---
+
+# STS Login
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-cloud">lakeFS Cloud</a>
+{: .label .label-green }
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-enterprise">lakeFS Enterprise</a>
+{: .label .label-purple }
+
+{: .note}
+> STS Login is available in lakeFS Cloud and lakeFS Enterprise. currently only tested on lakeFS Enterprise.
+
+Secure Token Service (STS) authentication in lakeFS enables users to authenticate to lakeFS using temporary credentials obtained from an Identity Provider (IdP) via the OpenID Connect (OIDC) Authentication workflow..
+This document outlines the process, from setting up the STS authentication flow to using temporary credentials to interact with lakeFS through the [high-level Python SDK](../../integrations/python.md).
+
+## Configuration
+
+To enable STS authentication, configure your lakeFS instance with the endpoint of the external Authentication Service.
+
+
+```yaml
+
+auth:
+    authentication_api:
+        endpoint: <url-to-remote-authenticator-endpoint>
+
+
+```
+
+- `auth.authentication_api.endpoint` `(string: https://external.authentication-service/api/v1)` - URL to external Authentication Service described at [authentication.yml](https://github.com/treeverse/lakeFS/blob/master/api/authentication.yml)
+
+Ensure you replace <url-to-remote-authenticator-endpoint> with the actual URL of your Authentication Service.
+
+
+## STS Login with high level python client
+
+The Python client simplifies initiating a client session with STS login. Use the following example to start a new session:
```

Comment:
Maybe can change this sentence to highlight that the high-level Python client is THE way to do STS login ?